### PR TITLE
Add column names to the result

### PR DESCRIPTION
**Pull request by vivekmenezes**

### CODE REVIEW

File: sql/sqlwire/sql_api.proto [hunk 1]
```diff
@@ -93,6 +93,9 @@ message SQLRequest {
 
 message SQLResponse {
```
Comment (tschottdorf) [inline sql/sqlwire/sql_api.proto]: in `:80`: s/consists/consist/

**Comment by tschottdorf** [comment]
LGTM

File: sql/sqlwire/sql_api.proto [hunk 2]
```diff
@@ -93,6 +93,9 @@ message SQLRequest {
...
   optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  // The names of the columns returned in the result set.
+  // The number of columns must equal the number of Datum in each Result.
```
Comment (mrtracy) [inline sql/sqlwire/sql_api.proto]: This is bordering on pedantic, but I don't see any mention here or on the other Response/Result types about how the columns in each row are ordered; I'm assuming there is a guaranteed order of columns shared by this "columns" set and the values in every `Result` message.

**Comment by mrtracy** [comment]
LGTM (with optional suggestion)

File: sql/sqlwire/sql_api.proto (hunk 2 above)
Comment (petermattis) [inline sql/sqlwire/sql_api.proto]: Columns are ordered by how they are requested in the SQL statement. And if ordering is unspecified there (e.g. `SELECT * FROM foo`) then they are ordered by how they were defined in the table. That is, `SELECT a, b FROM foo` will return the columns in the order `a`, `b`. And `SELECT b, a FROM foo` will return the columns in the order `b`, `a`. You can also do fun stuff like retrieve the same column multiple times: `SELECT a, a, a FROM foo`.

**Pull request merged**